src/awscli_login/config.py

```diff
@@ -77,7 +77,7 @@ class Profile:
             'factor': None,
             'passcode': None,
             'verbose': 0,
-            'refresh': 3000,  # in seconds (every 50 mins)
+            'refresh': 0,  # in seconds (every 50 mins)
```

Comment:
0 seconds probably doesn't translate to the 50 minutes mentioned in the comment. :-)

Reply:
I am approving despite the incorrect comment, but it would be really great if it were corrected before merging

Reply:
The comment is fixed.